var/spack/repos/builtin/packages/lcms2/package.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Lcms2(AutotoolsPackage):
```

Comment:
We already have an `lcms` package (which should arguably be renamed to `lcms2`). So you can remove this and use that instead.